### PR TITLE
don't request maintainer reviews if many files changed

### PR DESCRIPTION
**Pull request by LnL7**
This either indicates it's most likely either a tree-wide change,
staging-next merge or just a pull request with an incorrect target
branch.  For none of these cases are usually relevant for specific
package maintainers.

### CODE REVIEW

File: ofborg/src/tasks/eval/nixpkgs.rs [hunk 1]
```diff
@@ -270,6 +270,22 @@ impl<'a> NixpkgsStrategy<'a> {
                 },
             );
 
+            if changed_paths.len() > 64 {
```
Comment (cole-h) [inline ofborg/src/tasks/eval/nixpkgs.rs]: Would this be a good thing to make `const`? e.g. `const MAX_CHANGED_PATHS: u32 = 64;`

Additionally, is there a reason 64 was chosen? Or is it just an arbitrary number?

File: ofborg/src/tasks/eval/nixpkgs.rs [hunk 2]
```diff
@@ -270,6 +270,22 @@ impl<'a> NixpkgsStrategy<'a> {
...
+                info!(
+                    "pull request has {} changed files, skipping review requests",
+                    changed_paths.len()
+                );
+                let status = CommitStatus::new(
+                    self.repo.statuses(),
+                    self.job.pr.head_sha.clone(),
+                    String::from("grahamcofborg-eval-check-maintainers"),
+                    String::from("large change, skipping maintainer review"),
```
Comment (cole-h) [inline ofborg/src/tasks/eval/nixpkgs.rs]: I like the description from the `info!` above.

```suggestion
                    String::from("large change, skipping review requests"),
```

Maybe also `s/large/massive/`? ("mass rebuild", "massive change")

File: ofborg/src/tasks/eval/nixpkgs.rs (hunk 1 above)
Comment (LnL7) [reply ofborg/src/tasks/eval/nixpkgs.rs]: Mostly arbitrary, half is already seems rather large so I'm thinking this probably won't skip any cases that would be valid.

File: ofborg/src/tasks/eval/nixpkgs.rs [hunk 3]
```diff
@@ -270,6 +272,22 @@ impl<'a> NixpkgsStrategy<'a> {
                 },
             );
 
+            if changed_paths.len() > MAINTAINER_REVIEW_MAX_CHANGED_PATHS {
+                info!(
+                    "pull request has {} changed files, skipping review requests",
```
Comment (cole-h) [inline ofborg/src/tasks/eval/nixpkgs.rs]: Nit; might as well stay consistent with the variable's name.

```suggestion
                    "pull request has {} changed paths, skipping review requests",
```

**Review by cole-h — APPROVED** [review]

**Pull request merged**